Resources/Prototypes/Corvax/Recipes/Reactions/drinks.yml

```
@@ -0,0 +1,159 @@
+      amount: 2
+    BlueCuracao:
+      amount: 1
+    Absinthe:
+      amount: 1
+  products:
+    BadTouch: 6
+
+- type: reaction
+  id: DarkAndStormy
+  reactants:
+    Rum: 
```

yamllint

[error] trailing spaces

(trailing-spaces)